CMakeLists.txt

```diff
@@ -17,6 +17,15 @@
 #
 
 cmake_minimum_required(VERSION 2.8.12)
+
+# https://cmake.org/cmake/help/latest/policy/CMP0011.html
+# Fix warnings in CMake >= 3.18.0. OLD is deprecated.
+cmake_policy(SET CMP0011 NEW)
+# https://cmake.org/cmake/help/latest/policy/CMP0012.html
+# We require this for FindPython3 to work with CMake 3.18.2, but this will also generate warnings in
```

Comment:
What about:
"Setting the CMP0012 policy to NEW is required for FindPython3 to work with CMake 3.18.2 (bug in this particular version).
Otherwise, setting the CMP0012 policy is required for CMake versions >= 3.18.3 and a warning is generated if it is not the case."
Again, I am not completely sure about what is actually deprecated: not setting the policy or the OLD value of the policy, both ?

Reply:
Again, its both.